addons/flag-menu/userscript.js

```diff
@@ -0,0 +1,83 @@
+export default async function ({ addon, console, msg }) {
+  let greenFlag;
+
+  const contextMenuClass =
+    addon.tab.scratchClass("context-menu_context-menu") || addon.tab.scratchClass("context-menu_context-menu-content"); // React 16 || React 18
```

Comment:
Scratch updated React to 18 with the Face Sensing update, so this can be simplified. The style of the context menu was also changed, so some things may need to be tweaked.

Reply:
I mean, Scratch's new context menu has no transition for fading in/out, it just appears immediately, plus there's arrow-key navigation and the width was reduced. Every other property applies automatically.